prisma-fmt/tests/code_actions/scenarios/relation_mode_referential_integrity/result.json

```diff
@@ -10,8 +10,8 @@
             "character": 4
           },
           "end": {
-            "line": 4,
-            "character": 0
+            "line": 3,
+            "character": 35
```

Comment:
What does thi change mean?

Reply:
This is due to the changes in how we handle newlines with spans. This is the result of moving newlines _out_ of being included in the definition of a value. So now the diagnostic correctly represents the end of the _current_ line instead of the start of the _next_ line